ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V108_add_deduplication_info_to_completions.sql

```diff
@@ -0,0 +1,19 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+-- Add columns to store command deduplication information in completions.
+
+ALTER TABLE participant_command_completions
+    -- The application ID has to be provided by the application.
+    ADD COLUMN application_id text NOT NULL,
```

Comment:
The H2 and Oracle append-only index databases already have an `application_id`. Are you sure that the PostgreSQL index DB doesn't have this column already?

Reply:
8079673